setup.py

```diff
@@ -29,6 +29,12 @@
 if os.getenv("SNOWFLAKE_IS_PYTHON_RUNTIME_TEST", False):
     REQUIRED_PYTHON_VERSION = ">=3.8"
 
+PANDAS_REQUIREMENTS = (
```

Comment:
Is it parenthesis for formatting, or actually it's a tuple? I guess PANDAS_REQUIREMENTS should just be a list here?

Reply:
It's just for formatting. There's no comma after the list, so it's just a list, not a tuple of a list. the pre-commit hook likes to format it like this.

Reply:
hmm if I read correctly, isn't a comma after this list here (line 35)?

Reply:
Fixed.